### DISCUSSION TITLE
How does the PR selection and release process work in here?

### DISCUSSION THREAD

**Discussion by alasdaircs** [RicoSuter/NSwag#4384] [Q&A]
Is there any predictability about which PRs will make it into which release and when? The CONTRIBUTING doc provides no guidance in this respect and the [wiki Contribute page](https://github.com/RicoSuter/NSwag/wiki/Contribute) just guides people who want to help generally but not those with a PR ready.

There are currently 131 PRs outstanding. Some are bot-generated, and of the others some have been approved and some not. 

I'm happy for signposting answers if I've simply missed where this is documented.

**Comment by DanielLaberge** [comment]
I would like to know as well. This project seems interesting, but sadly it looks like it's not actively maintained (looking at the number of open issues and PRs and actual commits).

Before taking a dependency on this project, I'd like to know that any PRs we'd need to submit to fix issues will actually get merged.

If new maintainers are needed, an announcement would be appreciated. People might step up if so invited!

**Reply by DanielLaberge** [reply]
@RicoSuter Your input would be greatly appreciated, as it appears you may be the sole (active) person authorized to merge Pull Requests.

I understand that maintaining open source projects can be demanding, hence my proposal to consider allowing new maintainers to approve PRs. This might greatly benefit the project's longevity. Thank you.

**Reply by alasdaircs** [reply]
My PR Took about six weeks to get merged into the code and released as [v13.19.0](https://github.com/RicoSuter/NSwag/releases/tag/v13.19.0) . It's Open Source, and I guess Rico has to make a living. It's not dead, but no longer his focus. It looks like he just dips in and does a bit now and again. See my history: https://github.com/RicoSuter/NSwag/pull/4374.
